release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1174,6 +1174,17 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-release-note-node-tuning-operator-bug-fixes_{context}"]
 ==== Node Tuning Operator (NTO)
 
+* Previously, if you specified a long string of isolated CPUs in a performance profile, such as `0,1,2,...,512`, the `tuned`, Machine Config Operator, and `rpm-ostree` components failed to process the string as expected. As a consequence, after you applied the performance profile, the expected kernel arguments were missing. The system failed silently with no reported errors. With this release, the string for isolated CPUs in a performance profile is converted to sequential ranges, such as `0-512`. As a result, the kernel arguments are applied as expected in most scenarios. (link:https://issues.redhat.com/browse/OCPBUGS-45264[*OCPBUGS-45264*])
++
```

Comment:
Just wondering if putting the link under the note would create a closer relationship with the 45264 note?

Reply:
Probably leave it where it is. When it's after the note it look a bit detached from the note as a whole